### DISCUSSION TITLE
Get Full path file with Q-file after @input

### DISCUSSION THREAD

**Discussion by thiagoreri** [quasarframework/quasar#10517] [General - Components / Directives / etc]
Hello,

How to get the full file path using Q-file after choosing the file?

**Comment by ontwikkelfabriek** [comment]
@thiagoreri 

> How to get the full file path using Q-file after choosing the file?

You don't, Quasar apps for web(SPA/SSR) you will not be able to get full native path. Quasar apps in Cordova/Capacitor/Electron mode will be able to get full path names because you'll have access to the native file system.

**Reply by thiagoreri** [reply]
This is not possible with SPA?

**Reply by hawkeye64** [reply]
It's a security issue implemented by the browser. Check the MDN spec. This happens for any browser-based web app.
